cpu_scheduling_algorithms/rr_scheduling.cpp

```diff
@@ -0,0 +1,89 @@
+
+// Round Robin is a scheduling algorithm.
+// In Round Robin each process is assigned a fixed time slot in a cyclic way.
+// https://en.wikipedia.org/wiki/Round-robin_scheduling
+#include <iostream>
+#include <vector>
+#include <numeric>
+#include <string>
+#include <functional>
+
+/*----------------------------------------------------------------
+// Calculate the waiting times of a list of processes that have a
+//        specified duration.
```

Comment:
This should be used instead for the Doxygen documentation.
```suggestion
/**
 * @brief Calculate the waiting times of a list of processes that have a
 * specified duration.
 * 
```